### PR TITLE
Update & Fix

### PR DESCRIPTION
**Pull request by Henry2o1o**

### CODE REVIEW

File: res/values-de/derpquest_strings.xml [hunk 1]
```diff
@@ -467,6 +470,8 @@
     <string name="incall_vibration_category">Vibration bei eingehendem Anruf</string>
     <string name="incall_vibrate_connect_title">Vibriert bei Verbindung</string>
     <string name="incall_vibrate_call_wait_title">Vibriert beim Anklopfen von Anrufen</string>
+    <string name="incall_vibrate_call_wait_title">Vibrieren bei Anklopfen</string>
```
Comment (NurKeinNeid) [inline res/values-de/derpquest_strings.xml]: reminder for myself

File: res/values-de/derpquest_strings.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
     <string name="theming_title">Design</string>
     <string name="animations_title">Animationen</string>
     <string name="nav_bar_category_title">Navigationsleiste</string>
-    <string name="edge_lighting_category_title">Edge lighting</string>
+    <string name="edge_lighting_category_title">Randbeleuchtung</string>
```
Comment (NurKeinNeid) [inline res/values-de/derpquest_strings.xml]: reminder for myself

File: res/values-de/derpquest_strings.xml [hunk 3]
```diff
@@ -148,20 +146,21 @@
 
     <!-- Wakeup options -->
     <string name="wakeup_when_plugged_unplugged_title">Aufwecken beim Laden</string>
-    <string name="wakeup_when_plugged_unplugged_summary">Aufwachen des Gerätes, wenn Ladegerät angeschlossen/abgezogen wird.</string>
+    <string name="wakeup_when_plugged_unplugged_summary">Aufwechen des Gerätes, wenn Ladegerät angeschlossen/abgezogen wird.</string>
```
Comment (NurKeinNeid) [inline res/values-de/derpquest_strings.xml]: reminder for myself

**Comment by idoybh** [comment]
Still need this open guys?

**Comment by NurKeinNeid** [comment]
Yes, I need it

**Comment by NurKeinNeid** [comment]
squashed it here DerpLab/platform_packages_apps_DerpQuest@04f62ff260b963108d0995d4c9f1fce541a9a700
will merge to main branch if it builds fine :)

**Pull request closed without merging**